src/test/java/Tree/BinaryTreeTest.java

```diff
@@ -0,0 +1,82 @@
+package Tree;
+
+import org.junit.Test;
+import org.junit.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class BinaryTreeTest {
+
+    @Test
+    public void testInOrder() {

```

Comment:
You should test `inOrder` on a basic `Binary Tree`, not on a `BinarySearchTree`.